src/backend/partaj/core/views/common.py

```diff
@@ -40,13 +40,15 @@ def get(self, request, referral_id):
         api_note_request = NoteApiRequest()
 
         try:
-            referral_answer = get_object_or_404(
-                ReferralAnswer,
-                state=models.ReferralAnswerState.PUBLISHED,
-                referral__id=referral_id,
-            )
-
-            api_note_request.post_note(referral_answer)
+            referral_answer = ReferralAnswer.objects.filter(
+                state=models.ReferralAnswerState.PUBLISHED, referral__id=referral_id
+            ).last()
+
+            if referral_answer:
```

Comment:
dans un cas comme ça, privilégie le test de la condition inverse. Soit :

if not referral_answer:
    return response _=> d'ailleurs même si cette vue va sauter ajouter un message pour pouvoir debug plus faciulement_

api_note_request.post_note(referral_answer)